app/service/data_svc.py

```diff
@@ -131,3 +136,23 @@ def __init__(self, dao):
               'ON a.ability_id=b.id ' \
               'WHERE a.op_id = %s;' % op_id
         return await self.dao.raw_select(sql)
+
+    def _make_uuid(self, _filename):
+        uuid_string = _filename.split('/')[-1].split('.')[0]
+        try:
+            val = UUID(uuid_string, version=4)
+            return _filename
+        except ValueError:
+            _uuid = str(uuid4())
```

Comment:
if _uuid is only used on 149 then no need to create a separate variable just assign it directly to the ability id on 149

Reply:
It's used on 147 also